src/Umbraco.Infrastructure/Persistence/Repositories/Implement/DistributedJobRepository.cs

```diff
@@ -90,6 +90,7 @@ public void Delete(DistributedBackgroundJobModel distributedBackgroundJob)
     private DistributedJobDto MapToDto(DistributedBackgroundJobModel model) =>
         new()
         {
+            Id = model.Id ?? 0,
```

Comment:
```suggestion
            Id = model.Id,
```

If it doesn't need to be nullable, this update is needed too.